.gitignore

```diff
@@ -1,2 +1,5 @@
 bin/
-obj/
\ No newline at end of file
+obj/
+.vs/
+Icon Sources/
```

Comment:
This feels like a personal thing, not something that should be in the template

Reply:
The .vs/ thing kills me every time. I had to fork the mod-template. :)